From 411eb0d0f105ae0289753e0243a97410d4be9748 Mon Sep 17 00:00:00 2001
From: Jerome Brunet <jbrunet@baylibre.com>
Date: Tue, 24 Jan 2017 15:12:31 +0100
Subject: [PATCH 63/93] clk: gxbb: put dividers and muxes in tables

Until now, there was only 1 divider and 1 mux declared for the gxbb
platform. With the ongoing work on audio this is about to change
Use the same approach as gates for dividers and muxes, putting them
in tables to fix the register address at runtime.

Signed-off-by: Jerome Brunet <jbrunet@baylibre.com>

Conflicts:
	drivers/clk/meson/gxbb.c
---
 drivers/clk/meson/gxbb.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/drivers/clk/meson/gxbb.c b/drivers/clk/meson/gxbb.c
index d1dac0f..e17903b 100644
--- a/drivers/clk/meson/gxbb.c
+++ b/drivers/clk/meson/gxbb.c
@@ -954,12 +954,16 @@
 	&gxbb_mali_0_sel,
 	&gxbb_mali_1_sel,
 	&gxbb_mali,
+	&gxbb_mpeg_clk_sel,
+	&gxbb_sar_adc_clk_sel,
 };
 
 static struct clk_divider *gxbb_clk_dividers[] = {
 	&gxbb_mpeg_clk_div,
 	&gxbb_mali_0_div,
 	&gxbb_mali_1_div,
+	&gxbb_mpeg_clk_div,
+	&gxbb_sar_adc_clk_div,
 };
 
 struct gxbb_composite_clk {
-- 
1.9.1

